### DISCUSSION TITLE
AirSim on Real Drone Documentation

### DISCUSSION THREAD

**Discussion by Gastastrophe** [microsoft/AirSim#4359] [Support Q&A]
Is there somewhere we can read more about how to get AirLib deployed on a real drone? I tried to follow the tutorial here (https://microsoft.github.io/AirSim/custom_drone/) but MavLinkTest wasn't recognized as a command. I found main.cpp in the DroneServer folder that says it should demo how to get this running, but the imports failed on it. I've seen people talking in posts that seem to have this working. How can DroneServer be deployed?

I have an NVIDIA Jetson which will be onboard a drone (I don't have many details) that uses a Pixhawk flight controller. I am trying to run DroneServer and DroneShell on that computer and use some camera APIs to control the drone through an autonomous driving script powered by YOLOv3 object detection and tail-chasing navigation.

**Comment by thomasTUK** [comment]
Did it work?

**Reply by Gastastrophe** [reply]
No. I still have no idea how this can be done. I may need to build the drone controller myself and abandon Airsim at this rate.
